pkgs/zigtools/zls/pkg.yaml

```diff
@@ -1,5 +1,7 @@
 packages:
-  - name: zigtools/zls@0.14.0
+  - name: zigtools/zls@0.15.0-dev.129+bb6d6473
+  - name: zigtools/zls
+    version: 0.14.0
```

Comment:
```suggestion
  - name: zigtools/zls@0.14.0
  - name: zigtools/zls
    version: 0.15.0-dev.129+bb6d6473
```

We update the first element periodically to test new versions.
aqua gets the latest version from GitHub Releases.
aqua can't get dev versions.
So the version of first element should be the latest GitHub Release, and dev version should be pinned.